src/patternfly/components/DualListSelector/examples/DualListSelector.md

```diff
@@ -33,12 +33,12 @@ cssPrefix: pf-c-dual-list-selector
       {{/dual-list-selector-tools-actions}}
     {{/dual-list-selector-tools}}
     {{#> dual-list-selector-status}}
-      {{#> dual-list-selector-status-text}}
+      {{#> dual-list-selector-status-text dual-list-selector-status-text--id=(concat dual-list-selector-pane--id '-status-text')}}
```

Comment:
Instead of doing this in each HBS block, could we just generate the status text `id` and `aria-labelledby` directly in the HBS partial? We could do that by setting this in the status text partial (beware, untested code ahead)

`id="(concat dual-list-selector--id '-' dual-list-selector-pane--id '-status-text')"`

and this in the panel partial

 `aria-labelledby="(concat dual-list-selector--id '-' dual-list-selector-pane--id '-status-text')"`